dashboard/app/controllers/plc/user_course_enrollments_controller.rb

```diff
@@ -3,12 +3,8 @@ class Plc::UserCourseEnrollmentsController < ApplicationController
   authorize_resource only: :create
 
   def index
-    if user_course_enrollment_params[:course]
-      course_name = user_course_enrollment_params[:course].tr('-', '_').titleize
-      @user_course_enrollments = [@user_course_enrollments.find_by(user: current_user, plc_course: ::Course.find_by_name(course_name).try(:plc_course))]
-    else
-      @user_course_enrollments = @user_course_enrollments.where(user: current_user) if @user_course_enrollments
-    end
+    @user_course_enrollments = @user_course_enrollments.where(user: current_user) if @user_course_enrollments
+    render 'index', locals: {user_course_enrollments: @user_course_enrollments}
```

Comment:
I dont think so? I believe this scenario we're meant to show all the plc user course enrollments for this user.